dbt/redshift/redshift_etl.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import absolute_import
+import logging
+import os
+import psycopg2
+import subprocess
+from dateutil import rrule
+
+from parsely_raw_data import redshift as parsely_redshift
+from parsely_raw_data import utils as parsely_utils
+from dbt.redshift.settings.default import *
```

Comment:
It's better to import things by name, in case the names in that module shadow things in this one. It also makes it easier to figure out where functions/items etc used here can be located, if they are not defined in this file. I'm guessing this is mostly the constants used in the `SETTINGS_ARG_MAPPING` defined below?

Reply:
good point - and it's ok to list them all out? i was worried it would be too long of a list

Reply:
That's pretty normal for Python, to have a long list of imports. "explicit is better than implicit" etc etc.